### ISSUE TITLE
Provide a func to conveniently (stressed) get a list of all available datasets

### ISSUE THREAD

**Issue by Alvant** [machine-intelligence-laboratory/TopicNet#100]
or to get some very simple data information.

Because current `get_info` func:
```python
from topicnet.dataset_manager import api

api.get_info()
```
is expected to be used only in Jupyter notebook (and I have doubts if at least someone will be able to do it the right way right away).

What I am talking about, is that it would be nice to have a func like this:
```python
from topicnet.dataset_manager import api

api.get_datasets()  # ['postnauka', 'ruwiki_good', ...]
```
or make the aforementioned `get_info` be usable in terminal without this clumsy-dumpy HTML display stuff.

TL;DR We need a way to quickly and conveniently observe all the available datasets.

**Comment by Alvant** [comment]
Indeed, it would be nice to have it, but no one will do it (just as, most likely, no one will use it).
